build/images/ovs/build.sh

```diff
@@ -92,6 +92,16 @@ pushd $THIS_DIR > /dev/null
 
 OVS_VERSION=$(head -n 1 ../deps/ovs-version)
 
+BUILD_TAG=""
+case "$DISTRO" in
+    "ubuntu")
+        BUILD_TAG="22.04-ovs$OVS_VERSION"
+        ;;
+    "ubi")
+        BUILD_TAG="ovs$OVS_VERSION"
+        ;;
+esac
```

Comment:
Better to add a line here as well to print the BUILD_TAG:
`echo "BUILD_TAG: $BUILD_TAG"`

Reply:
done